src/operator/rnn-inl.h

```diff
@@ -21,7 +21,7 @@
  * Copyright (c) 2015 by Contributors
  * \file rnn-inl.h
  * \brief
- * \author Sebastian Bodenstein, Shu Zhang
+ * \author Sebastian Bodenstein, Shu Zhang(shu.zhang@intel.com)
```

Comment:
revert the change. This email is unavailable now.

Reply:
fixed